src/Elm/Syntax/Exposing.elm

```diff
@@ -58,7 +58,7 @@ type TopLevelExpose
 {-| Exposed Type
 -}
 type alias ExposedType =
-    { name : String
+    { name : Node String
```

Comment:
Isn't this a breaking change?

Reply:
Yes, that is the breaking change I mentioned in the description. I had a quick look at removing it, but I'm not yet familiar enough with the code and didn't succeed. I'll try again later.

Reply:
Oops, my bad. Easier to read PRs after a good nights sleep as well!